src/aotcompile.cpp

```diff
@@ -1197,11 +1204,13 @@ static void construct_vars(Module &M, Partition &partition) {
                                         GlobalVariable::ExternalLinkage,
                                         fidxs, "jl_fvar_idxs");
     fidxs_var->setVisibility(GlobalValue::HiddenVisibility);
+    fidxs_var->setDSOLocal(true);
```

Comment:
I don't think we need to annotate these dso_local, they're removed by multiversioning later anyways.